.gitmodules

```diff
@@ -1,3 +1,9 @@
 [submodule "test/unit-test/CMock"]
 	path = test/unit-test/CMock
 	url = https://github.com/ThrowTheSwitch/CMock
+[submodule "test/cbmc/aws-templates-for-cbmc-proofs"]
+	path = test/cbmc/aws-templates-for-cbmc-proofs
+	url = https://github.com/awslabs/aws-templates-for-cbmc-proofs.git
+[submodule "test/cbmc/litani"]
```

Comment:
Let's talk about which version of the starter kit you are using, I haven't merged the need for litani into the starter kit yet, that was going to be done in two weeks.

Reply:
We opportunistically added litani given example from other libraries and the version of the starter kit is: https://github.com/awslabs/aws-templates-for-cbmc-proofs/tree/ded3c0c483b5cb799498b071f718d6fd8723e448

Should we remove it?